docs/MS Word/Табуляция.md

```diff
@@ -0,0 +1,38 @@
+## Табуляция в MS Word — это отступ от начала строки до первого слова в тексте, а необходима она для того, чтобы выделить начало абзаца или новой строки. Функция табуляции, доступная в текстовом редакторе от Майкрософт по умолчанию, позволяет сделать эти отступы одинаковыми во всем тексте, соответствующими стандартным или ранее установленным значениям.
```

Comment:
[ch-md-error-first-line-h1](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-first-line-h1)